src/main/java/version1/gameUtil/widgets/inputfields/MazeTextField.java

```diff
@@ -0,0 +1,11 @@
+package version1.gameUtil.widgets.inputfields;
+
+import version1.gameUtil.listeners.implementations.AuthenticationManager;
+import version1.gameUtil.listeners.interfaces.LoginBListenerIF;
+
+public class MazeTextField extends AbstractMazeTextField {
+    private LoginBListenerIF mediator;
```

Comment:
Same here, should the declared type be MazeTextFieldIF ?

Reply:
Got it, thanks!